.github/workflows/ci.yaml

```
@@ -0,0 +1,77 @@
+            # Build number is a unique number for each build , which is used to tag the image
+            # aakash1707/mernspace-auth-service:build-${build_number} , this build number is provided by github
+            IMAGE_TAG: build-${{ github.run_number }}
+        # Only run this job if the push is to the main branch , we are only considering the main branch for deployment
+        if: github.ref == 'refs/heads/main' && github.event_name == 'push'
+        steps:
+            - name: Checkout Repository
+              uses: actions/checkout@v3
+              # checkout the code from the repository
+              # and login to docker hub
+            - name: Log into Docker Hub
+              uses: docker/login-action@v1
```

actionlint (1.7.7)

the runner of "docker/login-action@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)